src/einsteinpy/utils/christoffel.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from sympy import Matrix, diff, eye, sin, symbols, zeros
+
+
+def christoffels(list2d, syms):
+    """
+    Function to calculate christoffel symbols of a given metric
+
+    Parameters
+    ----------
+    list2d : list
+        2d list (Matrix) representing metric, containing ~sympy expressions
+    syms : list
+        1d list containing representaion of [x0,x1,x2...] in ~sympy expressions
+
+    Returns
+    -------
+    a : list
```

Comment:
It returns `christlist` if I am not wrong :)

Reply:
I would replace it with simply 
"""
Returns
--------
list
"""